packages/components/src/public/files/models.ts

```diff
@@ -0,0 +1,23 @@
+import {InferDomainResponse} from "../..";
```

Comment:
Since this `InferDomainResponse` type is used as a prop type for the `FileGridPreviewProps` variables, do you think we need to make this public as well?

Reply:
InferDomainResponse is now public in file InferDomainResponse.ts. Two functions that were in InferDomainResponse.ts, inferDomainFromFile() & getServerFilePreview(), now moved to internal/components/assay/utils.ts